### DISCUSSION TITLE
Sentinel 1 angle of incidence

### DISCUSSION THREAD

**Discussion by jgomezdans** [microsoft/PlanetaryComputer#304] [Data request]
Hello, 
It is quite important to have access to the Sentinel 1 angle of incidence (or view zenith angle) layer. This is different from the "local angle of incidence" that you use for the RTC correction. In Google's EarthEngine, this is stored (spatially subsampled) in the "angle" layer (see [here](https://developers.google.com/earth-engine/datasets/catalog/COPERNICUS_S1_GRD#description)). I guess it will be present in the products, but is there some way to extract it directly?

**Comment by TomAugspurger** [comment]
Thanks for opening this up.

>  I guess it will be present in the products, but is there some way to extract it directly?

I'm not sure :/

**Comment by jgomezdans** [comment]
I think I have now fixed this. The data is available in the XML metadata files in the original GRD product (but note #313). I think my code is probably harder than what it needs be (ideally, the angle of incidence should be an additional layer in the STAC), but it works, and we have checked the angles with independent estimates from SNAP.

The code for this is in [this gist](https://gist.github.com/jgomezdans/c245d3f3c98243a8b5ce422a5ca1c148), together with a demo notebook.

Ultimately, I think this is a sticking plaster, and I look forward to the incidence angle being made a first class STAC citizen ;-)
